app/init.php

```diff
@@ -1587,6 +1609,10 @@ function (mixed $value) {
     return getDevice(APP_STORAGE_UPLOADS . '/app-' . $project->getId());
 }, ['project']);
 
+App::setResource('deviceForSites', function ($project) {
+    return getDevice(APP_STORAGE_SITES . '/app-' . $project->getId());
```

Comment:
Should we store sites build on a separate storage? Might make thing unnecessarily complex for the edge. If deployments and build are using the same collection for sites and functions so we can use the same storage for all compute.

Reply:
Notice devices are same, just use different paths. Thanks to that, we faced no issues on executor. I expect the same worry-free support on edge.

Reply:
Luke mentioned edge already knows about site and function, so he is not worired.